index.html

```diff
@@ -0,0 +1,123 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>Cereal</title>
+	<link rel="stylesheet" type="text/css" href="quiz.css">
+</head>
+<body>
+	<article class="holec">
+		<header> 
+			<h1>Honey Bunches of Oats with Almonds</h1>
+		</header>
+		<sectiona>
+			<section class="firstsec"><img src="honeybunches.jpg" width= "350" height="350"></section>
```

Comment:
Two things on the `img`...

Don't use width and height attributes, use CSS to set the style. Also, all `img`s must have an `alt` attribute